modules/extensions-support/src/test/java/org/mule/module/extension/internal/metadata/extension/MetadataOperations.java

```diff
@@ -98,12 +106,14 @@ public Object outputAndMetadataKeyCacheResolver(@Connection MetadataConnection c
         return null;
     }
 
-
+    // TODO MULE-9572 Empty @MetadataScope for overriding inherited resolvers
+    @MetadataScope
```

Comment:
not really, without this the Output Metadata could return, for example, a PersonType. We wither declare here a NullMetadataResolver (that's what the empty annotation is doing), or create an OutputResolver that returns NullMetadataType, so internally we default to JavaType, or we return ObjectType in a the new Resolver.